src/policy/rbf.cpp

```diff
@@ -47,6 +47,29 @@ RBFTransactionState IsRBFOptInEmptyMempool(const CTransaction& tx)
     return SignalsOptInRBF(tx) ? RBFTransactionState::REPLACEABLE_BIP125 : RBFTransactionState::UNKNOWN;
 }
 
+bool IsRBFOptOut(const CTransaction& txConflicting)
```

Comment:
Hm, I think everything in util/rbf should be moved to policy/rbf actually :thinking: There's no reason to have two places for RBF (I think) and organization-wise, all RBF logic and code is policy.

Reply:
Not sure, it would make all users of policy/rbf depend on txmempool. Eg not sure the standalone binaries `wallet-util` and friends depend on it although they use util/rbf.

Reply:
Ah you're right @darosior